packages/services/video-tools/video_utils/ai_commands/video_commands.py

```
@@ -0,0 +1,661 @@
+        txt_file = output_path.with_suffix('.txt')
+
+        # Save JSON if format allows
+        if format_type in ['describe-video', 'json']:
+            with open(json_file, 'w', encoding='utf-8') as f:
+                json.dump(result, f, indent=2, ensure_ascii=False)
+            print(f"💾 Saved JSON: {json_file.name}")
+
+        # Save TXT if format allows
+        if format_type in ['describe-video', 'txt']:
+            with open(txt_file, 'w', encoding='utf-8') as f:
+                f.write(f"Analysis Result\n")
```

Ruff (0.14.11)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)